### DISCUSSION TITLE
I can't fork somehing

### DISCUSSION THREAD

**Discussion by vader9595** [community/community#14883] [Other Feature Feedback, Questions, & Ideas]
I tried to fork something and it says 'no more forks can be created. view existing forks.", can someone help?

![forkError](https://user-images.githubusercontent.com/102444679/163241085-e47ea19f-8cce-4b9d-8165-73974cd267a4.png)

**Comment by pps83** [comment]
@github-staff please fix this extremely annoying regression. Previously, attempts to fork a repo that was already forked would open existing fork. Now, it gets stuck at that irritating misleading window. Please revert to original behavior.

**Comment by trasherdk** [comment]
Currently, if unlucky, you have no way to click your way from an upstream to your fork.

Clicking `fork` shows me this:
![image](https://user-images.githubusercontent.com/5003891/163488448-a9ab55b2-a006-44c4-b74f-bb1159c9e561.png)
A message not really relevant to the situation.

Clicking `View existing forks.` shows this:
![image](https://user-images.githubusercontent.com/5003891/163488596-dd59551e-b38d-4154-9184-67ab2a370abc.png)

And my fork is not in the list of forks shown.

**Comment by sffej** [comment]
"No more forks can be created" - Yes this is a confusing message please revert code to redirect to existing fork :)

**Comment by Krugo-D** [comment]
How is this still an issue?

![image](https://user-images.githubusercontent.com/85778179/188934618-b3ef8781-6053-4ccd-bd70-6b85df7d9825.png)

**Comment by donnerhacke** [comment]
Problem still exists and prevents to provide small and easy to integrate pull requests for the same project concerning different, independent issues.

@github-staff Do you really want large, incomprehensive contributions, which need to be separated by the maintainers manually?
